framework/framework.go

```diff
@@ -81,9 +87,6 @@ func New(config Config) (*Framework, error) {
 	if config.ResourceRouter == nil {
 		return nil, microerror.Maskf(invalidConfigError, "config.ResourceRouter must not be empty")
 	}
-	if config.RestClient == nil {
-		return nil, microerror.Maskf(invalidConfigError, "config.RestClient must not be empty")
-	}
```

Comment:
There is still the name validation in the wrong block.

Reply:
Sorted.